src/Options/OptionsInterface.php

```diff
@@ -41,6 +42,7 @@ interface OptionsInterface {
 	public const TOURS                                     = 'tours';
 	public const UPDATE_ALL_PRODUCTS_LAST_SYNC             = 'update_all_products_last_sync';
 	public const WP_TOS_ACCEPTED                           = 'wp_tos_accepted';
+	public const CUSTOMER_DATA_TERMS                       = 'customer_data_terms';
```

Comment:
We try to keep these in alphabetical order for simplicity. Also if it's specifically for the Ads account, does it make sense to prefix it with `ADS_` so it's together with all the other Ads settings?